### DISCUSSION TITLE
Best theme for current site

### DISCUSSION THREAD

**Discussion by prosgarz35** [GetPublii/Publii#1620] [Help]
Hi there guys. Can you tell me which themes (free or not) i can use to make site on publii?
Current site is - https://kps-t.ru/
Thanks for any help :)

**Comment by udokan** [comment]
Publii is the best CMS for a blog, but I'm not sure it's suitable for such purposes. For example, you can create a better blog on Publii than [Lebedev's](https://blog.tema.ru/). The free [Tinkoff](https://www.tbank.ru/business/website-builder/) website builder is more suitable for your purposes.
